api/desecapi/tests/test_rrsets_bulk.py

```diff
@@ -48,6 +48,7 @@ def test_bulk_post_my_rr_sets(self):
         self.assertRRSetsCount(response.data, self.data)
 
     def test_bulk_post_rr_sets_empty_records(self):
+        # TODO discuss if we should instead of given an error just discard empty-record rr sets
```

Comment:
I think for `POST`, this should be considered a client error (as well as for `PUT`). Both are supposed to present complete RRset objects for publication (the difference being that the RRset does not need to be new in the `PUT` case).

I think that most people would assume this behavior, and it also helps debugging stuff on the client side. If people don't want to care about it, they can use `PATCH`.